docroot/themes/custom/mass_theme/templates/content/node--contact-information--map.html.twig

```diff
@@ -71,30 +71,38 @@
  *
  */
 #}
-{% include "@molecules/google-map.twig" with {
-  "googleMap": {
-    "static": true,
+{% include "@molecules/map-leaflet.twig" with {
+  "leafletMap": {
+    "isStatic": false,
```

Comment:
```suggestion
    "isStatic": true,
```
The scroll bug has not been fixed on mobile. Turning off interactions for the initial release https://jira.mass.gov/browse/DP-22029